components/automate-ui/e2e/event-feed-page.e2e-spec.ts

```diff
@@ -7,22 +7,6 @@ describe('Event Feed: Guitar String Graph Interaction', () => {
     browser.get('/event-feed');
```

Comment:
Didn't this URL change in this PR? 🤔

Reply:
Oh I see the only test left in this file is pending. Maybe just remove this file then?

Reply:
I removed this file.